pkg/apis/application/v1alpha1/types.go

```diff
@@ -950,8 +950,8 @@ type RepositoryCertificate struct {
 	CertSubType string `json:"cipher" protobuf:"bytes,3,opt,name=cipher"`
 	// Actual certificate data, protocol dependent
 	CertData []byte `json:"certdata" protobuf:"bytes,4,opt,name=certdata"`
-	// Certificate fingerprint
-	CertFingerprint string `json:"certfingerprint" protobuf:"bytes,5,opt,name=certfingerprint"`
```

Comment:
should the JSON names be Java case? I.e. `json:certFingerprint` rather than `json:certfingerprint`

Reply:
You're right, they should be camel cased to improve readability. However, changing the names now would also require changing some code in the UI, because some of the variable names in the UI (in the parts interfacing with the API) are interdependent with the JSON names. So I see two options:

- Change the JSON names now and also bring in the required UI changes to this PR
- Have a separate PR shortly after this has been merged which changes case on the variable names in types.go and the UI parts. Personally, I think this option would be the better one.

Which one do you prefer?

Reply:
lets do separate PR for the name change